lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-card-container {
+  margin: $nx-spacing-md 0;
+}
+
+.nx-card {
+  background-color: #fff;
+  border-radius: $nx-border-radius;
+  box-sizing: border-box;
+  margin-bottom: $nx-spacing-md;
+  padding: $nx-spacing-l $nx-spacing-xxl;
+}
+
+.nx-card__call-out {
+  .nx-icon--card-call-out {
+    font-size: 48px;
+  }
+
+  .nx-icon--card-call-out-xl {
+    font-size: 64px;
+  }
+}
+
+.nx-card__call-out {
```

Comment:
Can be merged with the block above.

Reply:
> Can be merged with the block above.

https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/3369c5413e09c9c3129815857eda15c90fe845d3